examples/nextjs/README.md

```diff
@@ -1,12 +1,14 @@
 # Grafbase ⨯ Next.js
 
-[Join our Discord](https://discord.com/invite/dYBUPpmV4K)
+[Join our Discord](https://discord.gg/grafbase)
 
+[![Deploy to Grafbase](https://grafbase.com/button)](https://grafbase.com/new/configure?template=Todo&source=https%3A%2F%2Fgithub.com%2Fgrafbase%2Fgrafbase%2Ftree%2Fmain%2Ftemplates%2Ftodo)
+</br>
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/grafbase/grafbase/tree/main/examples/nextjs) 
 
 ## Mandatory configuration
 
-You must rename your `.env.local.example` to `.env` and fill all of its properties.
+Rename your `.env.local.example` to `.env` and fill all the variables.
```

Comment:
```suggestion
Rename your `.env.example` to `.env` and fill all the variables.
```

Perhaps rename the file `.env.local.example` so nobody makes the mistake of removing just `.example` as `.env.local` probably won't work out of the box with `-r dotenv/config`.

I made this mistake yesterday.

Reply:
Done, yeah, right